tap_github/tap.py

```diff
@@ -26,7 +26,10 @@ class TapGitHub(Tap):
     config_jsonschema = th.PropertiesList(
         th.Property("user_agent", th.StringType),
         th.Property("metrics_log_level", th.StringType),
+        # Authorization options
         th.Property("auth_token", th.StringType),
+        th.Property("auth_tokens", th.ArrayType(th.StringType)),
+        th.Property("rate_limit_buffer", th.ArrayType(th.IntegerType)),
```

Comment:
SDK based taps and target can now provide inline documentation and then use that to print new documentation for a README.md file.

Here inline, for `auth_token` and `auth_tokens`, I think it would be helpful to note that they are mutually exclusive, and to describe the use cases for each.

```py
        th.Property("auth_token", th.StringType, description="..."),
        th.Property("auth_tokens", th.ArrayType(th.StringType), description="..."),
        th.Property("rate_limit_buffer", th.ArrayType(th.IntegerType), description="..."),
```

This makes them break into multi-line declarations once formatted, but the advantage to readability and inline expectation-setting is worth it, I think.

The readme docs for those settings can then be auto-generated with:

```
tap-github --about --format=markdown
```